pkg/profiling/task/network/analyze/layer7/protocols/http1/analyzer_test.go

```diff
@@ -243,19 +245,19 @@ func TestBuildHTTP1(t *testing.T) {
 
 	for _, testCase := range tests {
 		//t.Run(testCase.name, func(t *testing.T) {
-		analyzer := NewHTTP1Analyzer().(*HTTP1Analyzer)
+		analyzer := NewHTTP1Analyzer().(*Analyzer)
 		l := list.New()
 		var events = make([]struct {
 			start, end int
 		}, 0)
 		for _, event := range testCase.events {
-			req, resp := analyzer.buildHTTP1(l, &SocketDataUploadEvent{
-				DataID:   uint64(event.dataID),
-				MsgType:  base.SocketMessageType(event.dataType),
-				Sequence: uint16(event.sequence),
-				Finished: uint8(event.finished),
-				Buffer:   bufferConvert(event.data),
-				DataLen:  uint16(len(event.data)),
+			req, resp := analyzer.buildHTTP1(l, &base2.SocketDataUploadEvent{
+				DataID:       uint64(event.dataID),
+				MsgType:      base.SocketMessageType(event.dataType),
+				Sequence:     uint16(event.sequence),
+				FinishStatus: uint8(event.finished),
```

Comment:
I think `finished` is a better status name. FinishStatus means `finish the status`

Reply:
Updated.